README.md

```diff
@@ -54,7 +54,9 @@ feature not yet included in a release, make sure you're using Go 1.17+ and run
 the following:
 
 ```sh
-go get -v github.com/Shopify/kubeaudit
+go install -v github.com/Shopify/kubeaudit/cmd@latest
+mv ~/go/bin/cmd ~/go/bin/kubeaudit
```

Comment:
According to `go help install` (go version 1.19.3)

> Executables are installed in the directory named by the GOBIN environment
variable, which defaults to $GOPATH/bin or $HOME/go/bin if the GOPATH
environment variable is not set. Executables in $GOROOT
are installed in $GOROOT/bin or $GOTOOLDIR instead of $GOBIN.

So I don't think this command works reliably, since `install` could place the binary in one of several locations. I wonder if we should do some combination of `git clone` and `go build -o`?